schemas/context/player-state.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/player-state",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Player state information",
+  "type": "object",
+  "meta:status": "experimental",
+  "meta:extensible": true,
+  "description": "Information related to player state.",
+  "definitions": {
+    "player-state": {
+      "properties": {
+        "xdm:playerStateSet": {
+          "title": "Player state set",
+          "type": "boolean",
+          "description": "Whether or not the player state is set on that state."
+        },
+        "xdm:playerStateCount": {
+          "title": "Player state count",
+          "type": "number",
+          "description": "The number of times that player state was set on the stream."
+        },
+        "xdm:playerStateTime": {
+          "title": "Show type",
+          "type": "number",
```

Comment:
Should that be an integer?  What about a min/max?

Reply:
Done